toolkits/global/packages/global-forms/README.md

```diff
@@ -13,3 +17,265 @@ The `global-forms` component currently uses the `DEFAULT` brand only.
+                {
+                    "template": "text",
+                    "label": "Your name",
+                    "id": "your-name",
+                    "name": "your-name"
+                },
+                {
+                    "template": "text",
+                    "label": "Your email",
+                    "id": "your-email",
+                    "name": "your-email"
+                }
+            ]
+        }
+    ]
+}
+```
+
+If you do wish to include a legend, you can use HTML to style it and add semantic meaning. It is recommended to use a heading of the correct level as the legend content.
+
+```json
+"fieldsets": {
+    [
+        {
+            "legend": "<h2>My level 2 legend</h2>",
+            "fields": [
+                ...
+            ]
+        }
+    ]
+}
+```
+
+### Fields
+
+Fields take the `template` property to determine which of the `view/fields` template files they represent. This property along with `label`, `id`, and The `template` property sets the type of field - for example, `"template": "text"` renders a text input field. Aim to make the `template`, `label`, `id`, and `name` properties mandatory parts of your data schema.
+
+This component supports a wide range of standard form field attributes. So, for example, to include a `readonly` attribute on your text input, you  can include a property of the same name on the data:
+
+```json
+{
+    "template": "text",
+    "label": "Your email address",
+    "id": "your-email",
+    "name": "your-email",
+    "readonly": true
+}
+```
+
+The `hint` property adds hint text under the main label text but _inside_ the `<label>`. This means it is automatically available to screen reader software.
+
+```json
+"hint": "This will be the email address you used when registering"
+```
+
+The `optional` property adds _“(optional)”_ to the label text.
+
+```json
+"optional": true
+```
+
+### Errors
+
+Each field can have an `error` property. The inclusion of the property means the field is in an error state. The property value (a string) defines the error message the user sees.
+
+```json
+{
+    "template": "text",
+    "label": "Your name",
+    "id": "your-name",
+    "name": "your-name",
+    "error": "Enter your name"
+}
+```
+
+You can summarise errors using a top level `errorSummary` property (adjacent to the `fieldsets` property). Each error in the errors array must point to the `id` of the input it relates to and repeat its `error` message:
+
+```json
+"errorSummary": {
+    "id": "summary",
+    "title": "There are problems",
+    "errors": [
+        {
+            "id": "your-name-error",
+            "error": "Enter your name"
+        },
+        {
+            "id": "checkbox-terms-error",
+            "error": "Agree to the terms to continue"
+        }
+    ]
+}
+```
+
+### Making choices
+
+`Select` fields (using the `<select>` element) define their choices via an `options` property, which must be an array. The `selected` property is a Boolean:
```

Comment:
I think it's good to specify what "their choices" refer to. I know this is technical documentation, but the more we can do to contextualise it for designers now, the less we'll have to edit later when we layer in the design guidelines. 

```suggestion
`Select` fields (using the `<select>` element) define the user's answer choices with an `options` property, which must be an array. The `selected` property is a Boolean:
```